src/signals/incident/components/IncidentPreview/index.js

```diff
@@ -158,6 +178,13 @@ const IncidentPreview = ({ incident, preview }) => (
                 </Li>
               ))}
             </Ul>
+            <Hidden minBreakpoint="laptop">
```

Comment:
I wouldn't duplicate the control (and use Hidden) unless is strict necessary. I believe this is easy to solve through css. How about 
```
const LinkContainer = styled.div`
  padding-top: ${themeSpacing(5)};
  position:absolute;
  top: 0;
  right: 0;

  @media screen and ${breakpoint('max-width', 'laptop')} {
    position:inherit;
  }
`;
```

Reply:
Much better indeed, thanks for the suggestion!